topics/templates/document-attributes-product.adoc

```diff
@@ -11,6 +12,10 @@ ifeval::[{project_product_cd}==false]
 :project_name_full: Red Hat Single Sign-On
 :project_version: 7.3.0.GA
 :project_versionDoc: 7.3
+:project_previous_version: 7.2
```

Comment:
@stianst It was used in _topics/tutorials.adoc_:
```
[iankko@localhost openshift]$ grep -rHn 'project_previous_version' * | wc -l
6
```
but not that many times, it would deserve a dedicated asciidoc attribute. Will remove it.

Reply:
Dropped `project_previous_version` and refactored _Manual & Automatic DB Migration_ sections not to actually need it, see https://github.com/keycloak/keycloak-documentation/pull/496/commits/98cc71a8ba955182ff1629e1bc4719efe35597e8 for details.